src/org/labkey/test/tests/InlineImagesListTest.java

```diff
@@ -327,5 +336,51 @@ public final void testList() throws Exception
         }
+
+        log("Verify list archive export/import can round trip attachment successfully");
+        ManageListsGrid listsGrid = BeginPage.beginAt(this, getProjectName()).getGrid();
+        listsGrid.checkAllOnPage();
+        File listArchive = listsGrid.exportSelectedLists();
+
+        _containerHelper.createProject(IMPORT_PROJECT_NAME);
+
+        BeginPage.beginAt(this, IMPORT_PROJECT_NAME).importListArchive(listArchive);
+        _listHelper.goToList(LIST_NAME);
+
+        // Validate that list is imported as expected with attachments
+        assertElementPresent("Did not find the expected number of icons for images for " + LRG_PNG_FILE.getName(), Locator.xpath("//img[contains(@title, '" + LRG_PNG_FILE.getName() + "')]"), 1);
+        assertElementPresent("Did not find the expected number of icons for images for " + JPG01_FILE.getName(), Locator.xpath("//img[contains(@title, '" + JPG01_FILE.getName() + "')]"), 1);
+        assertElementPresent("Did not find the expected number of icons for images for " + PDF_FILE.getName(), Locator.xpath("//img[contains(@src, 'pdf.gif')]"), 1);
+        assertElementPresent("Did not find the expected text for " + PDF_FILE.getName(), Locator.xpath("//a[contains(text(), '" + PDF_FILE.getName() + "')]"), 1);
+
+        // Issue 53498: reject string attachment values for import
+        ImportDataPage listImportPage = _listHelper.clickImportData();
+        importFilePathError(listImportPage, "5", "absent.txt");
+        importFilePathError(listImportPage, "5", PDF_FILE.getName());
+        listImportPage.setCopyPasteMerge(false, true);
+        importFilePathError(listImportPage, "1", "absent.txt");
+        importFilePathError(listImportPage, "1", PDF_FILE.getName());
+        listImportPage.setCopyPasteMerge(true, true);
+        importFilePathError(listImportPage, "1", "absent.txt");
+        importFilePathError(listImportPage, "1", PDF_FILE.getName());
+        importFilePathError(listImportPage, "5", PDF_FILE.getName());
+    }
+
+    private void importFilePathError(ImportDataPage listImportPage, String key, String attachmentValue)
+    {
+        String pasteData = TestDataUtils.tsvStringFromRowMaps(List.of(Map.of(LIST_KEY_NAME, key, LIST_ATTACHMENT01_NAME, attachmentValue)),
+                List.of(LIST_KEY_NAME, LIST_ATTACHMENT01_NAME), true);
+        log(pasteData);
+        listImportPage.setText(pasteData);
+        listImportPage.submitExpectingError();
+        try
+        {
+            String expectedError = "Row 1: Can't upload '" + attachmentValue + "' to field " + LIST_ATTACHMENT01_NAME + " with type Attachment.";
+            isElementPresent(Locator.tagWithClass("div", "labkey-error").withText(expectedError));
+        }
+        catch(NoSuchElementException nse)
+        {
+            checker().fatal().error("Invalid attachment error not present.");
```

Comment:
Is this fatal? Seems like testing of the other import scenarios could still proceed (though will likely also fail).